### DISCUSSION TITLE
別傻了，AIGC 殺不死藝術家

### DISCUSSION THREAD

**Discussion by neil0258** [1101-GS3073-C/Python-Community-GS3073-C#672] [General]
AIGC 產生內容能取代插畫師、畫家的工作嗎？或更瘋一點，AI 能量產藝術作品，乃至根本上顛覆藝術市場嗎？AIGC 爆紅已和科技界探討許多，今天目光轉向從業者：AI 能否取代藝術家的工作？
畫家回答：下筆之前我自己都不知道會發生什麼事，AI如何知道？又怎麼算到？
藝術品的價值是感染力，而這只能來自人，AI作品太冷靜匠氣，「我看得出AI圖細節很精緻，但靈魂不統一，不夠動人。」
繪畫不僅考驗畫家技巧（這部分AI沒有問題），畫家身為人，感受、心情至成長歷程、教育背景等有影響，資料之龐大AI無法計算，真正創作過程在打破某些規則和邏輯。「我下筆前都不知道之後會發生什麼事，AI又怎麼知道，怎麼算出來？」插畫師說。
我的看法：雖然我沒有什麼藝術的底子，但我細看還是分辨得出AI作圖以及人工作圖的差別，某插畫師說明科班美術生結構、線條色彩和細節如何訓練，都因循和AI完全不同的邏輯。很多時候畫家只畫主觀感受到的東西，作品生動不表示和真實世界一模一樣。我認為，畫家這個職業應該是無法被AI智慧所取代的，畢竟人類是帶有情感的動物，常常夾雜著許多主觀情緒及自己內心的獨白，這是AI畫圖所無法帶來的共鳴！

https://technews.tw/2022/11/25/aigc-cant-kill-artists/

**Comment by bww0401** [comment]
我也同意藝術工作不會被取代，AI跑出來的肯定是很完美的作品，但真正能夠打動人心的創作，往往伴隨著不完美，所以真的能感動人的，還是人，透過資料蒐集所訓練的結果，太過客觀而冰冷

**Comment by 0212Jerry** [comment]
我也認為Ai 無法取代藝術家，因為藝術家是自發的想創作、想表達，然而Ai 並沒有辦法想創作而創作，而是為了創作而創作。藝術技巧即便重要層次與靈魂相比依舊相差甚遠。從這裡就可以看出機器與人本質上最大的差異，在情感這方面，人類依舊保有本性，而這本性只存在於人與人之間，並且難以複製。

**Comment by alvin1226** [comment]
我認為對AI來說，他是學習了大量的畫家的作品，在依照我們指定的主題來畫圖，換個方面來說我覺得AI是以他"自己"來創作，我認為這可以算是一種藝術創作了，畢竟也是他依照自己演算的結果來畫出
